src/components/implementation/cbuf_mid/mid_level/cbuf_mid.c

```diff
@@ -38,8 +38,8 @@ void cbuf_call(char c)
 	for (i = 0; i < iter ; i++){
 		int tmem;
 		cbt[i] = cbuf_null();
-		mt[i] = cbuf_alloc(SZ, &cbt[i]);
-		cbuf_unpack(cbt[i], &id, &tmem);
+		mt[i] = cbuf_alloc(SZ, &cbt[i], 1);
```

Comment:
I had the same thought about the API. Hiding the flags makes good sense to me so we don't have another API change when we put tmem's into generated code only.